### PR TITLE
Add 'git-commit-data-action' action

### PR DESCRIPTION
**Pull request by rlespinasse**
https://github.com/marketplace/actions/git-commit-data

Expose git commit information as environment variables

```yaml
- name: Expose git commit data
  uses: rlespinasse/git-commit-data-action@v1.x

- name: Print git commit data
  run: |
    echo "Get author info"
    echo " - ${{ env.GIT_COMMIT_AUTHOR }}"
    echo " - ${{ env.GIT_COMMIT_AUTHOR_NAME }}"
    echo " - ${{ env.GIT_COMMIT_AUTHOR_EMAIL }}"
    echo "Get committer info"
    echo " - ${{ env.GIT_COMMIT_COMMITTER }}"
    echo " - ${{ env.GIT_COMMIT_COMMITTER_NAME }}"
    echo " - ${{ env.GIT_COMMIT_COMMITTER_EMAIL }}"
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Generate Release Notes Based on Git References](https://github.com/metcalfc/changelog-generator)
 - [Enforce Policies on GitHub Repositories and Commits](https://github.com/talos-systems/conform)
 - [Auto Label Issue Based on Issue Description](https://github.com/Renato66/auto-label)
+- [Expose Git Commit Data As Environment Variables](https://github.com/marketplace/actions/git-commit-data)
```
Comment (jessicalostinspace) [inline README.md:138]: Please link the repository instead of the marketplace :)

Comment (rlespinasse) [reply README.md]: @jessicalostinspace the link is updated

**Comment by sdras** [comment]
This looks good, thanks!

**Comment by rlespinasse** [comment]
@sdras Thanks

**Pull request merged**